src/components/post-form/assets-form.tsx

```diff
@@ -0,0 +1,56 @@
+import { useContext } from "react";
+import { useQueryClient } from "react-query";
+
+import { PosterContext } from "@/context/poster";
+import { UploadDropzone } from "@/utils/uploadthing";
+
+import { AssetItem } from "../asset-item/asset-item";
+import { ScrollArea } from "../ui/scroll-area";
+
+export const AssetsForm = () => {
+  const queryClient = useQueryClient();
```

Comment:
What does queryClient do?

Reply:
`queryClient` is an object that allows us to interact with cache. It comes from the `react-query` package which also helps us manage the state when making queries (like obtaining the uploaded files, for example).

You can read more about `react-query` [here](https://tanstack.com/query/v4/docs/react/overview).